### PR TITLE
qt: Finetune Options Dialog

### PR DESCRIPTION
**Pull request by UdjinM6**
| Before | After |
|-|-|
| Screenshot 2020-09-15 at 00 49 29 | Screenshot 2020-09-15 at 00 49 03 |

Note: you can't see it on screenshots here but the long translation of the `connectSocksTor` checkbox text (e.g. in `ru`) also caused dialog resizing for bigger fonts (in case you are wondering why this one was tweaked too :) )

EDIT:
With this PR if a translation of the `connectSocksTor`  checkbox is too long it's simply clipped
Screenshot 2020-09-15 at 01 14 43
but you can see the whole text if you resize the dialog manually
Screenshot 2020-09-15 at 01 15 19
Basically, it's a sign that translation should be shortened somehow but it at least doesn't cause resizing issues like it does currently.

EDIT2: This PR also removes Window tab and moves all its options into Main tab, see https://github.com/dashpay/dash/pull/3709#issuecomment-692394887 for screenshots.

### CODE REVIEW

**Comment by UdjinM6** [comment]
One more tweak, thanks @xdustinface for the idea 👍 
| | Main | Window |
|-|-|-|
| Before 86f79f5 | Screenshot 2020-09-15 at 03 40 36 | Screenshot 2020-09-15 at 03 40 44 |
| After 86f79f5 | Screenshot 2020-09-15 at 03 40 20 |  |

(I disabled some code to make screenshots, there were no Window tab and there are no new options on Main tab on mac after the change)

**Comment by xdustinface** [comment]
Hmm im not really a fan of giving the dialog such a huge `min-width`.. adds a lot unused space if the translations aren't that long and font size is normal. What about clearing the checkboxes texts, place a normal label right besides them and put the text in there + activate `wordWrap` for the labels?

**Comment by UdjinM6** [comment]
@xdustinface how about 2669065279?

**Comment by xdustinface** [comment]
> EDIT: re "empty" checkboxes: I'd try to avoid hacks like that tbh

Yeah also not the biggest fan but if they don't give the checkbox `wordWrap` 🤷‍♂️ Other things just coming in my mind are either to resize the window when it gets opened with `QFontMetrics` based on the widest text in the dialog or just create a simple custom checkbox widget with a button for the checkbox and a label with wordwrap for the text and just use it all over the place in the UI. 

Anyway, im good with 6ce7634 if you don't want to spend more time on this 👍

**Comment by xdustinface** [comment]
One more for here: Thoughts about 5d3b1e2c44ea7db87b819caa87b4c02da268c1f9 i.e. dropping the surrounding "Expert" `QGroupBox` in `Options -> Wallet` to make the wallet tab consistent with the others? The "Expert" title isn't really needed, is it?

**Comment by UdjinM6** [comment]
Agree, this looks better 👍  Added.

**Review by xdustinface — APPROVED** [review]
ACK

**Review by PastaPastaPasta — APPROVED** [review]
ACK, looks good, however, please update the PR description to match new changes
